test/test_helper.bash

```diff
@@ -13,6 +13,15 @@ fixtures() {
   TEST_RELATIVE_FIXTURE_ROOT=$(bats_trim_filename "${TEST_FIXTURE_ROOT}" TEST_RELATIVE_FIXTURE_ROOT)
 }
 
+bats_sudo() {
+  local sudo_path=$(command -v sudo 2>/dev/null)
+  if [[ "$(whoami)" != 'root' ]] && [ -x "$sudo_path" ]; then
```

Comment:
I know that this stems from the original code but do we really want to fallback to no sudo when it is not executable and we are not root?

Reply:
I assumed that that was a way to support bespoke systems in which the current user is root-like but not actually root. I'm not sure it's worth supporting them.

Feel free to change it.